public/video-ui/src/util/video.js

```diff
@@ -99,17 +99,29 @@ export default class VideoUtils {
     return false;
   }
 
-  static getScheduledLaunch({ contentChangeDetails }) {
-    const scheduledLaunch = contentChangeDetails.scheduledLaunch;
-    return scheduledLaunch ? moment(scheduledLaunch.date) : null;
-  }
-
   static canUploadToYouTube({ youtubeCategoryId, channelId, privacyStatus }) {
     return !!youtubeCategoryId && !!channelId && !!privacyStatus;
   }
 
-  static getEmbargo({ contentChangeDetails }) {
-    const embargo = contentChangeDetails.embargo;
-    return embargo ? moment(embargo.date) : null;
+  static getScheduledLaunch(contentChangeDetails) {
```

Comment:
minor: we could use [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) here, then we can call this function with the video, e.g. `const scheduledLaunch = VideoUtils.getScheduledLaunch(this.props.video)`.